jupyterlab_pullrequests/manager.py

```diff
@@ -37,11 +37,16 @@ def post_file_comment(self, pr_id, filename, body):
 
     @gen.coroutine
     def get_file_nbdiff(self, prev_content, curr_content):
+        def read_notebook(content):
+            if content:
```

Comment:
This would be better as a guard clause:

```
if not content:
    return nbformat.v4.new_notebook()

return nbformat.reads(content, as_version=4)
```

e.g. special case the empty notebook rather than treating both cases equally.